docs/sql/statements/begin.rst

```diff
@@ -32,8 +32,9 @@ Description
 
 The statement starts a transaction block until it committed or rolled back.
 
-As CrateDB does not support transactions, this command has no effect and will
-be ignored.
+As CrateDB does not support transactions, the only effect of this command is
+to start a scope on which cursors ``WITHOUT HOLD`` can be
```

Comment:
```suggestion
to start a scope (transaction), in which cursors ``WITHOUT HOLD`` can be
```

Reply:
I changed "on" to "in", sorry about that.
Regarding "scope (transaction)" I worry it may get confusing because we are saying we do not support transactions, but I am happy to go with it if you think is clearer.